ios/brave-ios/Sources/Brave/Frontend/Shields/SubmitReportView.swift

```diff
@@ -108,21 +110,34 @@ struct SubmitReportView: View {
 
   @MainActor func createAndSubmitReport() async {
     let domain = Domain.getOrCreate(forUrl: url, persistent: !isPrivateBrowsing)
-
-    let report = WebcompatReporter.Report(
-      cleanedURL: url,
-      additionalDetails: additionalDetails,
-      contactInfo: contactDetails,
-      areShieldsEnabled: !domain.areAllShieldsOff,
-      adBlockLevel: domain.globalBlockAdsAndTrackingLevel,
-      fingerprintProtectionLevel: domain.finterprintProtectionLevel,
-      adBlockListTitles: FilterListStorage.shared.filterLists.compactMap({
-        return $0.isEnabled ? $0.entry.title : nil
-      }),
-      isVPNEnabled: BraveVPN.isConnected
+    let webcompatReporterAPI = WebcompatReporter.ServiceFactory.get(
+      privateMode: false
+    )!
+    webcompatReporterAPI.submitWebcompatReport(
+      reportInfo: .init(
+        channel: AppConstants.buildChannel.webCompatReportName,
+        braveVersion: String(
+          format: "%@ (%@)",
+          AppInfo.appVersion,
+          AppInfo.buildNumber
+        ),
+        reportUrl: url.absoluteString,
+        shieldsEnabled: String(!domain.areAllShieldsOff),
+        adBlockSetting: domain.globalBlockAdsAndTrackingLevel.reportLabel,
+        fpBlockSetting: domain.finterprintProtectionLevel.reportLabel,
```

Comment:
Similarly these 2 need the old extension on `ShieldLevel` from `WebcompatReporter.swift` (removed file).